lib/internal/module.js

```diff
@@ -130,3 +121,12 @@ function addBuiltinLibsToObject(object) {
     });
   });
 }
+
+module.exports = {
```

Comment:
Actually, I just realized we *do* need `exports` re-assigned here because `exports.requireDepth` is modified in this module.